packages/support/package.json

```diff
@@ -85,9 +85,6 @@
     "access": "public"
   },
   "types": "./build/lib/index.d.ts",
-  "devDependencies": {
```

Comment:
not sure this dev dependency is used anywhere

Reply:
or rather set the version manually